client/src/pages/automation/workflow-executions/WorkflowExecutions.tsx

```diff
@@ -191,126 +214,216 @@ const WorkflowExecutions = () => {
+                                                );
+                                            }}
+                                            value={filterProjectInstanceId}
+                                        />
+                                    </div>
                                 )}
 
                             {workflows?.length && (
-                                <ComboBox
-                                    items={workflows?.map((workflow) => ({
-                                        label:
-                                            workflow.label || 'undefined label',
-                                        value: workflow.id,
-                                    }))}
-                                    label="Workflow"
-                                    maxHeight
-                                    name="workflow"
-                                    onChange={(value) => {
-                                        if (value) {
-                                            setFilterWorkflowId(value.value);
-                                        } else {
+                                <div className="flex flex-col space-y-2">
+                                    <Label>Workflow</Label>
+
+                                    <ComboBox
+                                        items={workflows?.map((workflow) => ({
+                                            label:
+                                                workflow.label ||
+                                                'undefined label',
+                                            value: workflow.id,
+                                        }))}
+                                        maxHeight
+                                        name="workflow"
+                                        onChange={(item) => {
+                                            let workflowId;
+
+                                            if (item) {
+                                                workflowId = item.value;
+                                            }
+
                                             setFilterWorkflowId(undefined);
-                                        }
-                                    }}
-                                    value={filterWorkflowId}
-                                />
+                                            search(
+                                                filterStatus,
+                                                filterProjectId,
+                                                filterProjectInstanceId,
+                                                workflowId,
+                                                filterStartDate,
+                                                filterEndDate,
+                                                filterPageNumber
+                                            );
+                                        }}
```

Comment:
Jel se može ovo izvući u funkciju? Vidin da se ponavlja sa konzistentim varijacijama

Reply:
@kresimir-coko to ću tebi prepustiti